### PR TITLE
Form validation Demo: 'reset form' button added for mvc and netcore

### PR DESCRIPTION
**Pull request by AlexanderMoiseev**

### CODE REVIEW

File: MVCDemos/Views/Form/Validation.cshtml [hunk 1]
```diff
@@ -55,7 +56,13 @@
                     .Items(groupItems => {
                         groupItems.AddSimpleFor(m => m.Name);
                         groupItems.AddSimpleFor(m => m.Date);
-                    });
+                        groupItems.AddSimpleFor(m => m.VacationDates)
+                            .Editor(e => e
+                            .DateRangeBox()
```
Comment (ksercs) [inline MVCDemos/Views/Form/Validation.cshtml]: Are indents ok here?

File: MVCDemos/Views/Form/Validation.cshtml [hunk 2]
```diff
@@ -84,20 +91,36 @@
...
-
-                items.AddButton()
-                    .HorizontalAlignment(HorizontalAlignment.Left)
-                    .ButtonOptions(b => b.Text("Register")
-                        .Type(ButtonType.Success)
-                        .UseSubmitBehavior(true)
-                );
-
-            })
+                })
```
Comment (ksercs) [inline MVCDemos/Views/Form/Validation.cshtml:123]: Please check indents everywhere

File: MVCDemos/Views/Form/Validation.cshtml [hunk 3]
```diff
@@ -21,30 +22,30 @@
...
-                                .Buttons(buttons =>
-                                {
+                                .Buttons(buttons => {
                                     buttons.Add()
                                         .Name("password")
                                         .Location(TextEditorButtonLocation.After)
                                         .Widget(w => w.Button()
                                             .Type(ButtonType.Default)
-                                            .Icon(Url.Content("~/Content/Images/icons/eye.png"))
+                                            .Icon(Url.Content("eyeopen"))
```
Comment (Zedwag) [inline MVCDemos/Views/Form/Validation.cshtml]: I don't think we need `Url.Content` here anymore

File: MVCDemos/Views/Form/Validation.cshtml (hunk 1 above)
Comment (AlexanderMoiseev) [reply MVCDemos/Views/Form/Validation.cshtml]: nope )

**Review by Zedwag — DISMISSED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**